cmd/verify/app/repository.go

```diff
@@ -99,55 +157,62 @@ func verifyStagedMetadata(repository string) error {
 		if err = db.VerifySignatures(signed, name); err != nil {
 			if _, ok := err.(verify.ErrRoleThreshold); ok {
 				// we may not have all the sig, allow partial sigs for success
-				log.Printf("\tContains %d/%d valid signatures\n", err.(verify.ErrRoleThreshold).Actual, role.Threshold)
+				log.Printf("\tContains %d/%d valid signatures\n", err.(verify.ErrRoleThreshold).Actual, thresholds[name])
+				_, err := printAndGetSignedMeta(name, signed.Signed)
+				if err != nil {
+					return err
+				}
 			} else if err.Error() == verify.ErrNoSignatures.Error() {
-				log.Printf("\tContains 0/%d valid signatures\n", role.Threshold)
+				log.Printf("\tContains 0/%d valid signatures\n", thresholds[name])
```

Comment:
Can you add a code comment that this intentionally doesn't return an error or something along those lines, just to be clear that this is informational and not a failure?

Reply:
done!